cabby/geo/map_processing/edge.py

```diff
@@ -55,13 +53,13 @@ def from_projected(cls, u_for_edge, v_for_edge, length, highway, osmid, name):
     )
 
   @classmethod
-  def from_poi(cls, u_for_edge, v_for_edge, osmid):
+  def from_poi(cls, u_for_edge, v_for_edge, osmid, dist):
```

Comment:
nit: probably best to add Arguments, Returns to the docstring here

Reply:
Added.